### PR TITLE
Fixed parsing vCard message to send contact correctly

### LINKED ISSUE #2680: Not able to send Contact
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

I am trying to send contact in contact card formate. But not able to send.  

### Expected behavior

Want to Get contact in contact card formate.

### Steps to Reproduce the Bug or Issue

 1. when I am sending a contact it goes to text formate.

### Relevant Code

client.sendMessage(contactNo+ '@c.us',
        'BEGIN:VCARD\n' +
        'VERSION:3.0\n' +
        'N:Ln;abc;;;\n' +
        'FN:abcLn\n' +
        'TEL;type=Mobile;waid=917491945069:+91 74919 45069\n' +
        'END:VCARD',{
          parseVCards: false
      }

### Browser Type

Google Chrome

### WhatsApp Account Type

Standard

### Does your WhatsApp account have multidevice enabled?

Yes, I am using Multi Device

### Environment

OS: Window
Phone OS: [Android, iOS]
whatsapp-web.js version [1.23.1-alpha.0]
WhatsApp Web version [run await client.getWWebVersion()]:
Node.js Version [v18.18.0]

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by khsoh**
# PR Details

## Description

The current implementation in version 1.26.0 does not work because the _parseVcard_ function does not exist.
Fixes the handling for vCard message to send contact correctly

## Related Issue

https://github.com/pedroslopez/whatsapp-web.js/issues/2680

This PR replaces the non-working PR: https://github.com/pedroslopez/whatsapp-web.js/pull/1995

## Motivation and Context

This fix will enable sending of vCard properly

## How Has This Been Tested

By sending a vCard formatted message.

Test platform:
Ubuntu: 22.04.5 LTS
Node: 20.17.0
WhatsApp Web: 2.3000.1016908551

## Types of changes

- [ ] Dependency change
- [ x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] My code follows the code style of this project.
- [ ] I have updated the documentation accordingly (index.d.ts).

### CODE REVIEW

**Comment by khsoh** [comment]
Hi.  Is there a reason for closing this pull request?

**Comment by alechkos** [comment]
@khsoh Please fix eslint checks and also provide a code you used to test the PR

File: src/util/Injected/Utils.js [hunk 1]
```diff
@@ -135,11 +135,16 @@ exports.LoadUtils = () => {
         } else if (options.parseVCards && typeof (content) === 'string' && content.startsWith('BEGIN:VCARD')) {
             delete options.parseVCards;
             try {
-                const parsed = window.Store.VCard.parseVcard(content);
+                // eslint-disable-next-line no-useless-escape
+                const rgx_vcard = new RegExp('^(?:\\s*BEGIN:VCARD\\s*\\n\\s*VERSION:[2-9]\.[0-9](?=\\n))'
```
Comment (alechkos) [inline src/util/Injected/Utils.js:139]: We have `parseVcard` in the module `WAWebVcardParsingUtils`, please use it instead.

**Review by alechkos — CHANGES_REQUESTED** [review]

File: src/util/Injected/Utils.js (hunk 1 above)
Comment (khsoh) [reply src/util/Injected/Utils.js:139]: It was not working .  parseVcard function does not seem to be present.

You can try sending a VCard string via SendMessage to test:

let vcfmessage = 'BEGIN:VCARD\n' +
  'VERSION:4.0\n' +
  'FN:Tom Smith\n' +
  'N:Smith;Tom;;;\n' +
  'TEL;type=CELL;type=VOICE;waid=11234567890:+1 123 456 7890\n' +
  'END:VCARD';

client.sendMessage(number, vcfmessage);

The original library does not work.

**Comment by alechkos** [comment]
We are going to continue with #1995 since it uses the internal parser and was opened first.

**Pull request closed without merging**